.github/ISSUE_TEMPLATE/feature.yml

```
@@ -10,28 +10,26 @@ body:
         You'll need to demonstrate widespread support for your idea among the community.
 
         Major feature proposals should generally be discussed on [Discourse](https://discuss.python.org/c/ideas/6) before opening a GitHub issue. Wait until it's clear that most people support your idea before filling in this form.
-  - type: checkboxes
+  - type: dropdown
     attributes:
-      label: Has this already been discussed elsewhere?        
+      label: Has this already been discussed elsewhere?
       options:
-        - label: I have already discussed this feature proposal on Discourse
-        - label: This is a minor feature, which does not need previous discussion elsewhere
+        - No response given
+        - I have already discussed this feature proposal on Discourse
+        - This is a minor feature, which does not need previous discussion elsewhere
+      multiple: false
+    validations:
+      required: true
   - type: textarea
     attributes:
       label: "Links to previous discussion of this feature:"
     validations:
       required: false
-  - type: input
-    attributes:
-      label: "Summary of proposal:"
-      description: A one-line summary of your proposal.
-    validations:
-      required: true
   - type: textarea
     attributes:
-      label: "Pitch:"
+      label: "Proposal:"
       description: >
-        Explain why this feature or enhancement should be implemented and how it would be used.
+        Explain what your proposal is, why it should be implemented and how it would be used.
```

I think this reads better?:

```suggestion
        Explain your proposal, why it should be implemented, and how it would be used.
```

---

Much, thanks!